src/fsharp/FSharp.Build/Fsc.fs

```diff
@@ -116,57 +116,58 @@ type FscCommandLineBuilder () =
 type [<Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")>] Fsc() as this = 
     inherit ToolTask()
     let mutable baseAddress : string = null
+    let mutable capturedArguments : string list = []  // list of individual args, to pass to HostObject Compile()
+    let mutable capturedFilenames : string list = []  // list of individual source filenames, to pass to HostObject Compile()
     let mutable codePage : string = null
+    let mutable commandLineArgs : ITaskItem list = []
     let mutable debugSymbols = false
     let mutable debugType : string = null
     let mutable defineConstants : ITaskItem[] = [||]
+    let mutable delaySign : bool = false
     let mutable disabledWarnings : string = null
     let mutable documentationFile : string = null
+    let mutable dotnetFscCompilerPath : string = null
     let mutable embedAllSources = false
     let mutable embed : string = null
     let mutable generateInterfaceFile : string = null
+    let mutable highEntropyVA : bool = false
     let mutable keyFile : string = null
     let mutable noFramework = false
     let mutable optimize  : bool = true
-    let mutable tailcalls : bool = true
```

Comment:
@forki ... 
Yeah ... I hated having to look through the whole file to see if something was implemented.